exporters/trace/zipkin/zipkin.go

```diff
@@ -166,19 +166,12 @@ func (e *Exporter) ExportSpans(ctx context.Context, batch []*export.SpanData) er
 	if err != nil {
 		return e.errf("request to %s failed: %v", e.url, err)
 	}
-	e.logf("zipkin responded with status %d", resp.StatusCode)
+	defer resp.Body.Close()
 
-	_, err = ioutil.ReadAll(resp.Body)
-	if err != nil {
-		// Best effort to clean up here.
-		resp.Body.Close()
-		return e.errf("failed to read response body: %v", err)
+	if resp.StatusCode != 202 {
```

Comment:
This is enough for zipkin (and zipkin clones). We don't need to read the body as it does not return meaningful information more than the status code.